### DISCUSSION TITLE
Possible UI bug on github.com: No scroll/overflow content if I click on "Show more" under "Recent Repositories", Chrome Version 107.0.5304.88

### DISCUSSION THREAD

**Discussion by stephanietuerk** [community/community#38102] [Other Feature Feedback, Questions, & Ideas]
Just wanted to let you all know that if I click "Show more" under recent repositories I get a long list of repos (exactly what I wanted, great!) but unfortunately the list extends offscreen and there is no scrollbar on the element containing the list, so I cannot visually access them. (Was able to add scrollbar in dev tools to get what I needed.) 

![image](https://user-images.githubusercontent.com/13357861/199582373-f1440d07-a905-4928-8527-ed589ffcbd1a.png)
![image](https://user-images.githubusercontent.com/13357861/199582879-ca6dfeae-a528-47bf-b784-ac090b36c6f0.png)

**Comment by ossanna16** [comment]
Hi there @stephanietuerk and welcome to our community! Thank you for reporting this!

**Comment by brtnfld** [comment]
I second this, I have a lot of repositories listed under "Recent Repositories", the only way to scroll down the list is to use the scroll bar on the right, and that only starts scrolling the list once one has scrolled all the way through all the "Following" notices. Can a scroll bar be added to the frame listing the "Recent Repositories"? This was for firefox.
